.metals/metals.log

```diff
@@ -0,0 +1,8043 @@
+[0m2021.08.11 16:03:37 INFO  Started: Metals version 0.10.6-M1 in workspace '/Users/smashaal/me/code/nvim-treesitter' for client Neovim 0.6.0.[0m
```

Comment:
I don't think we need those files

Reply:
🤦 dammit I can't believe I accidentally pulled those in